integration-tests/zendeskRequestAPI.spec.ts

```diff
@@ -78,9 +84,45 @@ describe('Zendesk ticket check', () => {
       'X-Zendesk-Webhook-Signature': signature
     }
   }
+
+  let ticketId: string
+
+  beforeAll(async () => {
+    ticketId = await createZendeskRequest()
+    return ticketId
+  })
+
+  afterAll(async () => {
+    return deleteZendeskTicket(ticketId)
+  })
+
+  test('API Gateway returns 200 for a matching zendesk ticket', async () => {
+    const webhookRequestData = {
```

Comment:
Since you are using this a few times, you could declare a webhookRequestData interface and give it values in your tests.

Reply:
added !